### ISSUE TITLE
support for ruby version 2.6.4

### ISSUE THREAD

**Issue by nijeesh4all** [rbspy/rbspy#227]
i am getting this issue when i am running rbspy 

```
Profiler error: Rbspy failed with error:
thread '<unnamed>' panicked at 'Ruby version not supported yet: 2.6.4. Please create a GitHub issue and we'll fix it!', src/core/initialize.rs:333:14
```

ruby version :- 2.5.4
rails version :- 5.2.3
OS: Mac OS 10.45

```
$ rbspy --version
rbspy 0.3.7

```

**Comment by daniellockyer** [comment]
@nijeesh4all Adding this in with #229 :)

**Comment by daniellockyer** [comment]
Merged!

**Comment by alberto** [comment]
Thanks @daniellockyer. Do you intend to release a new version anytime soon?

**Comment by daniellockyer** [comment]
@alberto Will get one released after #232 gets in.

@jvns Do I need to publish a new version of `rbspy-ruby-structs` first? Would you be able to give me crates.io access?
